src/App.js

```diff
@@ -536,6 +538,21 @@ export default class App extends Component {
                           </AuthorizedElement>
                         }
                       />
+                      <Route
+                        path="Entities"
+                        element={
+                          <AuthorizedElement tokenDecoded={this.state.tokenData} iSuperAdmin={true}>
+                            <EntitiesPage
+                              token={this.props.accessToken}
+                              tokenData={this.state.tokenData}
+                              env={env}
+                              graphqlErrors={this.state.connectionIssue}
```

Comment:
why graphql?

Reply:
i'm

> why graphql?

i'm passing the state of the application to the page if a connection error needs to be displayed is going to be displayed